src/screens/NetworkSettings/CustomNetworkSettingsScreen.js

```diff
@@ -154,8 +154,11 @@ export const CustomNetworkSettingsScreen = ({ navigation }) => {
     setInvalidModel(validate(form));
   };
 
-  const handleFeedbackModalDismiss = () => {
+  const handleFeedbackModalDismiss = (navigate) => {
     dispatch(networkSettingsUpdateReady());
+    if (navigate) {
+      navigation.navigate('Dashboard');
```

Comment:
suggestion: Use `NavigationService.resetToMain();` instead, so that the "Back" button from OS does not return to the Custom Network screen.

See its usage on App.js for a reference.

https://github.com/HathorNetwork/hathor-wallet-mobile/blob/e0f165c58b428498622751b2f54d7215f4312150/src/App.js#L779-L783